lib/screens/register_screen.dart

```diff
@@ -81,13 +87,81 @@ class _RegisterScreenState extends State<RegisterScreen> {
     });
   }
 
+  void _showImagePickerOptions() {
+    showModalBottomSheet(
+      context: context,
+      builder: (BuildContext context) {
+        return Column(
+          mainAxisSize: MainAxisSize.min,
+          children: <Widget>[
+            ListTile(
+              leading: const Icon(Icons.photo_library),
+              title: const Text('Choose from Gallery'),
+              onTap: () {
+                _pickImage(ImageSource.gallery);
+                Navigator.pop(context);
+              },
+            ),
+            ListTile(
+              leading: const Icon(Icons.photo_camera),
+              title: const Text('Take a Photo'),
+              onTap: () {
+                _pickImage(ImageSource.camera);
+                Navigator.pop(context);
+              },
+            ),
+          ],
+        );
+      },
+    );
+  }
+
+  Future<void> _pickImage(ImageSource source) async {
+    final pickedImage = await ImagePicker().pickImage(source: source);
+
+    if (pickedImage != null) {
+      setState(() {
+        _imageFile = PickedFile(pickedImage.path);
+      });
+    }
+  }
+
   void submitSignUp() {
-    BlocProvider.of<CredentialCubit>(context).signUpSubmit(
-        user: UserEntity(
-      fullName: _fullNameController.text,
-      email: _emailController.text,
-      password: _passwordController.text,
-    ));
+    if (_imageFile != null) {
+      File imageFile = File(_imageFile.path);
+      di.sl<UploadProfileImageUseCase>().call(file: imageFile).then((imageUrl) {
+        BlocProvider.of<CredentialCubit>(context)
+            .signUpSubmit(
+                user: UserEntity(
+                    fullName: _fullNameController.text,
+                    email: _emailController.text,
+                    password: _passwordController.text,
+                    imgUrl: imageUrl))
+            .then((value) {});
```

Comment:
sama ini .then nya jan dibikin kosong Register profile successfully..dikasih snackbar ya

Reply:
sama yg disini diubah aja jadi Logged in successfully!! ya @ketrinvania